packages/core/src/tools/utils/urlPolyfill.spec.ts

```diff
@@ -66,3 +81,9 @@ describe('getHash', () => {
     expect(getHash('http://www.datadoghq.com/foo/bar?a=b#hello')).toBe('#hello')
   })
 })
+
+describe('getLinkElementOrigin', () => {
+  it('should retrieve the url origin if URL origin is "null"', () => {
+    expect(getLinkElementOrigin({ origin: 'null', host: 'foo.bar', protocol: 'file:' } as URL)).toBe('file://foo.bar')
+  })
+})
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Since we already have tests on `getOrigin`, could we test this case with `getOrigin`?